releasetasks/templates/l10n.yml.tmpl

```diff
@@ -0,0 +1,32 @@
+{% for platform in l10n_platforms %}
+# TODO: iterate over platform specific locales...probably need l10n json to do this
+{% for locale, locale_revision in l10n_changesets.iteritems() %}
+# TODO: make a helper function to generate consistent builder names?
+{% set buildername = "{}_{}_{}_l10n_repack".format(branch, product, platform) %}
+-
+    taskId: "{{ stable_slug_id(buildername) }}"
```

Comment:
Looks like you're heading towards one buildbot builder, with --locale <ab-CD> varying between instances. Do you need unique taskId's for each locale ?